Jenkinsfile

```
@@ -65,26 +64,26 @@ pipeline {
 
                 stage ('py36') {
                     steps {
-                        echo "Not yet implemented."
-                        // echo "Running nosetests on Python 3.6"
-                        // sh 'python3.6 -m pip install . -U --user'
-                        // sh 'python3.6 -m pip install nose_xunitmp --user'
-                        // sh "python3.6 setup.py nosetests --with-xunitmp --with-xcoverage --cover-package=${KATPACKAGE}"
+                         echo "Running nosetests on Python 3.6"
+                         sh 'python3.6 -m pip install . -U --user'
+                         sh 'python3.6 -m pip install nose_xunitmp --user'
+                         sh "python3.6 setup.py nosetests --with-xunitmp --with-xcoverage --cover-package=${KATPACKAGE} --with-xunit --xunit-file=nosetests_py36.xml"
                     }
                 }
             }
 
             post {
                 always {
-                    junit 'nosetests.xml'
+                    junit 'nosetests_*.xml'
```

@mmphego I removed this because tests failed when generating the reports in the Futurize stage 1 branch please do check it out.
```suggestion
                    junit 'nosetests.xml'
```